### ISSUE TITLE
with 1.11.1, builder.sh fails with "cannot stat ...dockerinit-1.11.1’: No such file or directory"

### ISSUE THREAD

**Issue by gcbartlett** [hypriot/rpi-docker-builder#14]
According to the docker 1.11 release notes:

- dockerinit is no more (#19490,#19851)

**Comment by szepeviktor** [comment]
See https://github.com/docker/docker/pull/19490

**Comment by szepeviktor** [comment]
@StefanScherer Would it be possible to update the project to Docker v0.12.x ?

**Comment by StefanScherer** [comment]
@szepeviktor There is no need for this repo. You can build Docker directly from docker/docker repo. 

```
git clone https://github.com/docker/docker
cd docker
make deb
```

Only Git, Make and Docker are needed on host machine.

**Comment by szepeviktor** [comment]
Thank you. 
For my platform (ThunderX arm64) there's no docker prebuilt.


-- 
+36204242498
Ezen a készüléken sok az elütés. 
Elnézést!

**Comment by StefanScherer** [comment]
Ok a `make deb` is on its way with pull request https://github.com/docker/docker/pull/27625 which builds 1.13-dev
Otherwise use `make binary` or `make dynbinary`.

**Comment by szepeviktor** [comment]
Thank you.
> For my platform (ThunderX arm64) there's no docker prebuilt.

Will `make dynbinary` run without Docker already installed?

**Comment by StefanScherer** [comment]
No, for all make targets you need Docker installed. Try using packet.net <http://packet.net/> they have a really big ARMv8 machine for 0,50$/hour. Use the apt-get install docker.io <http://docker.io/> for the build.

**Comment by szepeviktor** [comment]
Thank you again.

I've made the test for them, see the comments: https://www.packet.net/blog/arming-the-world-with-an-arm64-bare-metal-server/#dsq-app1

I am a Debian guy and was always using apt.dockerproject.org as Docker is not available in Debian.
In apt.dockerproject.org the package name is `docker-engine` which is not in Ubuntu.
